experiments/list_unsupported.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/sh
+
+# gnat2goto on the path?
 if ! command -v gnat2goto > /dev/null; then
   echo >&2 "gnat2goto not on PATH!"
+  gnat2goto_bin=$(cd "$(dirname ${0})/../gnat2goto/install/bin" 2>/dev/null && pwd)
```

Comment:
Really minor nitpick: This'll break if some crazy person decides to symlink this script. I'd expect a person doing this would also know how to use it already, so wouldn't need this help message, so I don't consider this an issue.

Reply:
Yeah.  `list_unsupported.sh` is intended as a development utility, not a final deliverable so this kind of robustness, although a good thought, is probably more than we need.

Reply:
Yeah, good catch - but I think this is good enough for now.